tuxemon/states/world/worldstate.py

```diff
@@ -620,6 +620,25 @@ def get_all_entities(self) -> Sequence[NPC]:
         """
         return list(self.npcs.values())
 
+    def check_collision_zones(
+        self,
+        map: Mapping[Tuple[int, int], Optional[RegionProperties]],
+        label: str,
+    ) -> Optional[Tuple[int, int]]:
+        """
+        Returns coords (tuple) of specific collision zones.
+
+        Returns:
+            The coordinates.
+
+        """
+        for coords, props in map.items():
+            if isinstance(props, Dict):
```

Comment:
This works, but strictly speaking should be dict, not Dict.

Reply:
True